vavedem-restapi/src/main/resources/application.yml

```diff
@@ -14,34 +22,34 @@ spring:
         max-pool: 10
         max-idle: 0
     jpa:
-        hibernate:
-            ddl-auto: create
-            temp:
-                use_jdbc_metadata_defaults: false
+        properties:
```

Comment:
capul properties trebuie scos iar "ddl-auto: create" trebie adaugat inapoi, altfel baza de date nu se mai genereaza automat in entitati